library/src/morpheus-master/js/src/sakai.morpheus.update.presence.js

```diff
@@ -1,4 +1,49 @@
-function updatePresence(){
+var active_tab_listener_attached = false; // initial load
+
+// Set the name of the hidden property and the change event for visibility
+var hidden, visibilityChange;
+if (typeof document.hidden !== "undefined") { // Opera 12.10 and Firefox 18 and later support 
+    hidden = "hidden";
```

Comment:
I'd just make hidden a boolean. You don't use that string in any console output, so there's no point overcomplicating it. Actually, forget that. I'd just use document.hidden. It's supported in all the browsers we're interested in. Sakai won't work in IE < 10, anyway. I'd just remove your hidden var and use document.hidden.

Reply:
I'm goign to remove the variable declaration - it's not used (should have seen that ;p) and it works without it.

Browser support for the Page Visibility API is currently good.
    IE - 10+
    Firefox - 10+
    Chrome - 14+
    Safari - 6.1+
    Opera - 12.1+

Reply:
Awesome browser support. Great.